core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -19,7 +19,7 @@ package org.apache.spark
 
 import java.io._
 import java.lang.reflect.Constructor
-import java.net.{MalformedURLException, URI}
```

Comment:
Removed unused import in this PR, because this one line change is not encouraged as a separate PR.